Makefile

```diff
@@ -678,6 +678,9 @@ CPP_PROTOS_CCL_TARGET := $(LOCAL_BIN)/.cpp_ccl_protobuf_sources
 .DEFAULT_GOAL := all
 all: $(COCKROACH)
 
+.PHONY: libs
+libs: $(C_LIBS_CCL)
```

Comment:
Can we call this `c-deps` for consistency with `clean-c-deps` and `c-deps-fmt`?

Reply:
Done.